androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/TripStops.kt

```diff
@@ -13,5 +32,107 @@ fun TripStops(
     stopSequence: Int?,
     headerSpec: TripHeaderSpec?,
     now: Instant,
-    global: GlobalResponse?
-) {}
+    global: GlobalResponse?,
+    routeAccents: TripRouteAccents
+) {
+    val context = LocalContext.current
+
+    val splitStops: TripDetailsStopList.TargetSplit? =
+        remember(targetId, stops, stopSequence, global) {
+            if (stopSequence != null && global != null) {
+                stops.splitForTarget(targetId, stopSequence, global, combinedStopDetails = true)
+            } else null
+        }
+
+    var stopsExpanded by rememberSaveable { mutableStateOf(false) }
+
+    val routeTypeText = routeAccents.type.typeText(context, isOnly = true)
+    val stopsAway = splitStops?.collapsedStops?.size
+    val target = splitStops?.targetStop
+    val hideTarget =
+        headerSpec is TripHeaderSpec.Scheduled &&
+            target != null &&
+            target == stops.startTerminalEntry
+
+    val showFirstStopSeparately =
+        when (headerSpec) {
+            TripHeaderSpec.FinishingAnotherTrip,
+            TripHeaderSpec.NoVehicle -> true
+            else -> false
+        }
+
+    val lastStopSequence = stops.stops.lastOrNull()?.stopSequence
+
+    Column {
```

Comment:
suggestion: I know UI isn't in scope for this ticket, but may be worth adding just the background color here since it is hard to read the text against the default route color background.

Reply:
Added.